MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+  grid.SetRow(dragIndicatorView, 1);
+  grid.SetColumn(dragIndicatorView, 0);
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+The screenshot shows the output of the reordering items by drag and drop. 
+
+![Item reordering by drag and drop](Images/item-drag-and-drop/maui-listview-dragviewindicator.gif)
+
+## Drag item customization
+
+By defining the `SfListView.DragItemTemplate` property of the `SfListView`, you can display a custom user interface (UI) when performing drag and drop operations. The template can be defined either in code or XAML.
+
+N> If the `BackgroundColor` is set for the `DragItemTemplate` or `DragIndicatorView`, make sure to set `InputTransparent` to `true`. This is necessary because dragging does not occur when using the `DragIndicatorView` in UWP.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnHold"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60">
+  <syncfusion:SfListView.DragItemTemplate>
+    <DataTemplate>
+      <Grid Padding="10">
+        <Label x:Name="textLabel" Text="{Binding Name}" FontSize="15" />
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.DragItemTemplate>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+listView.DragItemTemplate = new DataTemplate(() => {
+  var grid = new Grid();
+  var name = new Label { FontSize = 15 };
+  name.SetBinding(Label.TextProperty, new Binding("Name"));
+  grid.Children.Add(name);
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+## Event
+
+The `ItemDragging` event is raised while dragging and dropping the item in the `SfListView`. The `ItemDraggingEventArgs` has the following members which provides the information for the `ItemDragging` event:
+
+ * `Action`: Returns the drag `Action` such as start, dragging, and drop.
+ * `Bounds`: Return bounds of drag item when dragging and dropping.
+ * `Handled`: If this member is set to `true`, dragging can be handled. It is applicable only if `Action` is `Dragging`.
+ * `DataItem`: Returns the underlying data of the dragging item. 
+ * `NewIndex`: Returns the item index of the `DataSource.DisplayItems` where dragging item is going to be dropped.
+ * `OldIndex`: Returns the item index of the `DataSource.DisplayItems` where dragging item started. The OldIndex and NewIndex will be same if `Action` is `Start`.
+ * `Position`: Returns the touch position of the drag item from screen coordinates.
+
+## Auto scroll options
+
+### Auto scroll margin
+
+To adjust auto scroll margin, set a value to the `ScrollMargin` property of `AutoScroller` to enable auto-scrolling while dragging. The default value is `15`. Auto-scrolling will be enabled when reaching `ScrollMargin` from view bounds while dragging.
```

Comment:
To adjust the auto-scroll margin, set a value to the `ScrollMargin` property of `AutoScroller` to enable auto-scrolling while dragging. The default value is `15`. Auto-scrolling will be enabled when reaching the `ScrollMargin` from view bounds while dragging.

Reply:
Addressed the concerns provided.